yetanotherco/aligned_layer
# feat(operator): add aggregator rpc client

This PR adds the operators RPC client to communicate with the aggregator.

To test it:
* start anvil
* start aggregator with `make aggregator-start`
* start operator with `make operator-start`
* in another terminal, send a proof to verify `make send-plonk-proof`

You should see logs in both the operator and aggregator:
* **Aggregator**: `New Task response...`
* **Operator**: `Signed task response header accepted by aggregator.`
